contrib/cirrus/runner.sh

```diff
@@ -217,6 +257,11 @@ function _run_build() {
 }
 
 function _run_altbuild() {
+    if _only_changes_are_in docs hack test
+    then
+        return
+    fi
+
```

Comment:
Skipping `hack` here also makes me nervous.  Right below is a call to `hack/make-and-check-size` and I know for sure the Windows build depends on both both `hack` and `docs` :disappointed: